Makefile.am

```diff
@@ -14,4 +16,6 @@ chil_la_SOURCES = e_chil.c vendor_defns/hwcryptohook.h e_chil_err.h
 noinst_HEADERS = e_chil_err.c
 
 # Override the usual and make sure to install in OpenSSL's default engine store
-pkglibdir = $(libdir)/engines
+# If you are not building against a system-installed OpenSSL, point
+# the variable PKG_CONFIG_PATH to your lib/pkgconfig directory for the make install step
+pkglibdir = `pkg-config --variable=enginesdir libcrypto`
```

Comment:
This change of `pkglibdir` is probably a matter of debate.  Would you mind putting that in a different PR?